modules/ROOT/partials/commands/advtable-cmds.adoc

```diff
@@ -20,7 +21,10 @@
 | Sorts the current table descending by row based on the current cursor position or selection.
 
 | mceTableToggleSeries
-| Toggles a series column on the selected table. For details, see <<usingmcetabletoggleseries,Using `mceTableToggleSeries`>>. include::partial$misc/requires_5_9v.adoc[]
+| Toggles a series column on the selected table. For details, see xref:usingmcetabletoggleseries[Using `mceTableToggleSeries`].
+
+include::partial$misc/requires_5_9v.adoc[]
```

Comment:
```suggestion
| Toggles a series column on the selected table. For details, see xref:usingmcetabletoggleseries[Using `mceTableToggleSeries`].
include::partial$misc/requires_5_9v.adoc[]
```
This is part of the last cell in this row so we should keep it together.

Reply:
For me it renders like this:

<img width="801" alt="Screen Shot 2022-03-23 at 23 21 07" src="https://user-images.githubusercontent.com/38878982/159805829-18a7edf7-a66e-4644-a384-8084e2500e92.png">
 
Isn't this correct ?